dockers/docker-lldp/lldpmgrd

```diff
@@ -94,6 +94,11 @@ class LldpManager(daemon_base.DaemonBase):
         """
         port_desc = None
 
+        # Skip port name prefixed with "Inband". These are recycle ports exposed in PORT_TABLE for
+        # asic-to-asic communication in VOQ based chassis system. We do not configure LLDP on these.
+        if 'Inband' in port_name:
```

Comment:
Can you use `inband_prefix()` from sonic_py_common/interface.py?

Reply:
Yes. I'll fix this.

Reply:
Fixed.